### PR TITLE
refactor(assets): move image assets to `assets` package

### PR DESCRIPTION
**Pull request by montezume**
#### Summary

This PR moves all of the images from `@commercetools-frontend/ui-kit/images`, that are currently used in app kit, into a new package, `@commercetools-frontend/assets`. This way the files can be removed from `ui-kit`. 

#### Approach

Currently, in our webpack config, we are using `webpack-url-loader` to base64 incode SVG files from external sources. I'm doing the same here with a rollup plugin.

### CODE REVIEW

File: rollup.config.js [hunk 1]
```diff
@@ -53,6 +54,10 @@ const config = {
       preferBuiltins: true,
       modulesOnly: true,
     }),
+    url({
+      limit: 10 * 1024 * 100, // inline all png and svg files files (matches our webpack behaviour)
```
[the remark this answers is not in the document]
Comment (tdeekens) [reply rollup.config.js]: Sure. Makes sense to me.

**Comment by montezume** [comment]
Might make sense to include `'images'` in `files`, that way users who want to use the files directly and deal with their own build setup, can?

**Comment by montezume** [comment]
@emmenko I removed the images from `orders` ,`products` and `discounts` folders in the last commit.

**Comment by emmenko** [comment]
@montezume the svgs of "image missing/broken etc" are generic enough that can stay here. Sorry for not being clear enough 😇

**Pull request merged**